### PR TITLE
Dupaxes

### PR DESCRIPTION
**Pull request by jw3126**
I have a fortran code, which produces "AnatomicalOrientation = ???". This patch makes `MetaImageFormat.jl` accepts values of "AnatomicalOrientation" that would error before with a warning:
```
Warning: AnatomicalOrientation string unrecognized, got ???. Using xyz instead
```
I am no expert on .mhd however. It might be that "AnatomicalOrientation = ???" is a crazy thing to do in the first place, then it might not be a great idea to merge this.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+AxisArrays = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
+ColorVectorSpace = "c3611d14-8923-5661-9e6a-0046d554d3a4"
+Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MappedArrays = "dbb5928d-eab1-5f90-85c2-b9b0edb7c900"
+
+[compat]
+julia = "≥ 1.0.0"
```
Comment (timholy) [inline Project.toml]:
```suggestion
julia = "1"
```
It's safe to bet that most packages will be broken by Julia 2.0, so declaring that this only supports Julia 1 proactively ensures that the package system will install a version that is known to work.

**Review by timholy — APPROVED** [review]
I'm no expert on .mhd either, and in my estimation it's an incompletely specified format. So let's do the pragmatic thing and merge this.

**Comment by timholy** [comment]
Maybe also delete REQUIRE?

**Comment by jw3126** [comment]
Should I commit Manifest.toml or add to .gitignore?

**Comment by jw3126** [comment]
@timholy do you have an opinion on what to do with Manifest.toml? If not I will just add it to .gitignore and merge.

**Pull request merged**